samples/nrf9160/ciphersuites/README.rst

```diff
@@ -0,0 +1,146 @@
+.. _ciphersuites:
+
+nRF9160: TLS Ciphersuites
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The TLS Ciphersuites sample demonstrates a minimal implementation of a client application that attempts to connect
+to a host by trying different TLS ciphersuites. This sample will show that some ciphersuites are supported by the host, while others
+aren't, finally providing a summary of the support.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160_ns
+
+.. include:: /includes/spm.txt
+
+Overview
+********
+
+The sample first initializes the :ref:`nrfxlib:nrf_modem` and AT communications.
+Next, it provisions a root CA certificate to the modem using the :ref:`modem_key_mgmt` library.
+Provisioning must be done before connecting to the LTE network, because the certificates can only be provisioned when the device is not connected.
+
+The sample then iterates through a list of TLS ciphersuites, attempting connection to the host with each one of them.
+Ciphersuites that are supported by the host will connect successfully, while unsupported ciphersuites will cause a connection failure, setting ``errno`` to ``95``.
+
+Finally, the sample will end with a summary of the ciphersuites supported and unsupported by the host ``example.com``.
```

Comment:
```suggestion
Finally, the sample provides a summary of the cipher suites that the host support and does not support ``example.com``.
```

Reply:
Suggesting to move the sentence in line 146 up here and make it the last sentence in the overview section (after line 35).